dcos_launch/config.py

```diff
@@ -284,10 +285,27 @@ def _validate_genconf_scripts(genconf_dir, dcos_config):
                     dcos_config[filename_key], genconf_dir))
 
 
+def dcos_version_later_than_1_12(installer_url: str):
```

Comment:
I would perhaps make this more generic:
def dcos_version_check(installer_url: str, target_version: str) or somesuch where we pass the expected version as a parameter.